### PR TITLE
Remove acra-connector mentions

### PR DESCRIPTION
**Pull request by Lagovas**
* removed acra-connector mentions
* changed local specific hostnames to localhost for deployed tools
* updated example for psql connection using TLS
* updated descriptions of accessible services for all examples

### CODE REVIEW

**Review by vixentael — APPROVED** [review]
bye-bye acra connector

File: README.md [hunk 1]
```diff
@@ -656,7 +683,7 @@ These are all the code changes! 🎉
 
 Learn how to configure AcraCensor – SQL firewall – to allow or deny specific queries and make your application more steady against SQL injections.
 
-What's inside: OWASP Mutillidae vulnerable web application, AcraConnector, AcraServer, AcraCensor.
+What's inside: OWASP Mutillidae vulnerable web application, AcraServer, AcraCensor.
```
Comment (vixentael) [inline README.md:686]: have you updated our example with acracensor as well?

File: README.md [hunk 2]
```diff
@@ -587,15 +612,17 @@ Fields `authors`, `description` and `summary` are encrypted. So, the data is sto
 
 ### 2.5 Other available resources
 
-1. PostgreSQL – connect directly to the database using the admin account `rubygems`/`rubygems`: [postgresql://www.rubygems.example:5432](postgresql://www.rubygems.example:5432).
+1. PostgreSQL – connect directly to the database using the admin account `rubygems`/`rubygems`: [postgresql://www.rubygems.example:5432](postgresql://localhost:5432).
 
-2. Prometheus – examine the collected metrics: [http://www.rubygems.example:9090](http://www.rubygems.example:9090).
+2. pgAdmin - connect directly to the database using WebUI and user account `login:test@test.test`/`password:test`: [http://localhost:8008](http://localhost:8008)
```
Comment (shadinua) [inline README.md:617]:
```suggestion
2. pgAdmin - connect directly to the database using WebUI and user account login:`test@test.test`/password:`test`: [http://localhost:8008](http://localhost:8008)
```
Copy-paste will work better.

**Review by shadinua — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Lagovas) [reply README.md:686]: no. returned connector for connector

**Pull request merged**